stl/inc/chrono

```diff
@@ -5477,7 +5477,14 @@ namespace chrono {
 
             int _Estimated_width = -1;
             (void) _Measure_string_prefix(_Stream.view(), _Estimated_width);
-            return _Write_aligned(_STD move(_FormatCtx.out()), _Estimated_width, _Specs, _Fmt_align::_Left,
+
+            auto _FormatSpecs = _Specs;

```

Comment:
Naming nitpick: We conventionally use `_Ugly_snake_case` for variables, and `_UglyCamelCase` for template parameters. `<format>` and its related code are somewhat inconsistent (e.g. `_FormatCtx` violates this convention) but we should try to follow it in new code. This will improve consistency with the existing code you mentioned, where the name is `_Format_specs`:

https://github.com/microsoft/STL/blob/87580ca91c1a34645d22f79f7640a13f68bb31c5/stl/inc/format#L3639-L3643